pages/api/journal/get.ts

```diff
@@ -0,0 +1,18 @@
+import { NextApiResponse, NextApiRequest } from "next";
```

Comment:
I think this file should be renamed. This file gets blogs by review status and we'll have another one that gets blogs by a unique id.

Reply:
That's true, I didn't think about that. I'll fix that right way.